### PR TITLE
Deprecate omnibus-chef

### PR DESCRIPTION
**Pull request by ksubrama**

### CODE REVIEW

**Comment by ksubrama** [comment]
@jkeiser @btm @chefsalim @schisamo @danielsdeleo @tyler-ball 
/cc @chef/engineering-services @chef/omnibus-maintainers

**Comment by lamont-granquist** [comment]
sweet :+1:

**Comment by thommay** [comment]
heck yes! :+1:

**Comment by danielsdeleo** [comment]
:+1:

**Comment by scotthain** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,130 +1,32 @@
...
-### Build
-
-You create a platform-specific package using the `build project` command:
-
-```shell
-$ bundle exec omnibus build <PROJECT>
-```
+* (Chef Client omnibus directory)[https://github.com/chef/chef/tree/master/omnibus]
+* (ChefDK omnibus directory)[https://github.com/chef/chef-dk/tree/master/omnibus]
+* (Push Jobs omnibus directory)[https://github.com/chef/opscode-pushy-client/tree/master/omnibus]
```
Comment (schisamo) [inline README.md]: Let's call this _Push Jobs Client_

File: README.md [hunk 2]
```diff
@@ -1,130 +1,32 @@
...
-C:\Users\vagrant>cd C:\home\vagrant\omnibus-chef
-
-C:\home\vagrant\omnibus-chef>bundle install --without development
-
-C:\home\vagrant\omnibus-chef>bundle exec omnibus build <PROJECT> -l internal
-```
+(opscode-ci)[https://github.com/chef-cookbooks/opscode-ci] should contain the
+build pipeline configurations.
+(opscode-ci#366)[https://github.com/chef-cookbooks/opscode-ci/pull/366] is the
+PR that redirected all pipelines to build from their new sources.
```
Comment (schisamo) [inline README.md]: I would just leave this bit out as opscode-ci is not public.

**Comment by schisamo** [comment]
A few small comments but a huge :+1:!

**Comment by ksubrama** [comment]
Comments fixed!  Goodbye omnibus-chef!

**Pull request merged**